packages/manager/apps/container/src/container/common/language/Button.tsx

```diff
@@ -22,10 +22,13 @@ function LanguageButton({
   const isSmallDevice = useMediaQuery({
     query: `(max-width: ${SMALL_DEVICE_MAX_SIZE})`,
   });
+
   return (
     <button
-      aria-haspopup={show}
+      id="language-menu-button"
```

Comment:
is the id used somewhere ?

Reply:
it appears on test, but not sure it is used. Even if it is not mandatory by WCAG or RGAA I think we can keep it